### PR TITLE
[Android] Scroll to first unread message.

### PR DESCRIPTION
**Pull request by vishwesh3**
So that first unread message is visible.

Same as iOS.

### CODE REVIEW

File: src/message/MessageListContainer.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ class MessageListContainer extends PureComponent<Props> {
     const unreadMessageIds = filterUnreadMessageIds(visibleMessageIds, flags);
 
     if (unreadMessageIds.length > 0) {
-      queueMarkAsRead(auth, unreadMessageIds);
+      // queueMarkAsRead(auth, unreadMessageIds);
```
Comment (vishwesh3) [inline src/message/MessageListContainer.js]: Will uncomment once PR is completed (have to do some testing).

**Comment by borisyankov** [comment]
Nice that you are making progress on this.

Two major things to consider:
* the `autoscrollToBottom` does not need to be state just a field in the class
* you likely don't need the value at all. Just do scrollToBottom on content change?

**Comment by vishwesh3** [comment]
Yup! totally agree with both, will work on it more and remove state and extra possible. 👍

**Comment by vishwesh3** [comment]
Updated 👍 did some more improvements (but still not completed) and now first unread message is at top of the screen :)

**Comment by vishwesh3** [comment]
@borisyankov Ready for review :)
This PR scroll's to first unread message after narrowing. (there might be some issue in scroll position just after opening app, but for subsequent narrows it will scroll to first unread message, will solve this in another PR :))

**Comment by vishwesh3** [comment]
Updated :+1:
now it scrolls for first render too :)

**Comment by borisyankov** [comment]
Cool. Looks good to me.
I'll merge it and release a new Beta.
We might retire this code anyways, when the HTML version is ready, so let's not spend eternity polishing it (and it is good).

**Pull request merged**